docs/deployment/add-ons/load-balancer/README.md

```diff
@@ -0,0 +1,192 @@
+            - ```
+                export TAG_VALUE=<>
+                export CLUSTER_SUBNET_IDS=$(aws ec2 describe-subnets --region $CLUSTER_REGION --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$CLUSTER_NAME --output json | jq -r '.Subnets[].SubnetId')
+                for i in "${CLUSTER_SUBNET_IDS[@]}"
+                do
+                    aws ec2 create-tags --resources ${i} --tags Key=kubernetes.io/cluster/${CLUSTER_NAME},Value=${TAG_VALUE}
+                done
+                ```
+        1. `kubernetes.io/role/internal-elb`. Add this tag only to private subnet
+        1. `kubernetes.io/role/elb`. Add this tag only to public subnet.
+1. Load balancer controller will use [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)(IRSA) to access AWS services. An OIDC provider must exist for your cluster to use IRSA. Create an OIDC provider and associate it with for your EKS cluster by running the following command if your cluster doesn’t already have one:
+    1.  ```
+        eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} --region ${CLUSTER_REGION} --approve
+        ```
+1. Create an IAM role with [these permissions](../../../../awsconfigs/infra_configs/iam_alb_ingress_policy.json) for the load balancer controller to use via a service account to access AWS services.
+    1. ```
+        export LBC_POLICY_NAME=alb_ingress_controller_${CLUSTER_REGION}_${CLUSTER_NAME}
+        export LBC_POLICY_ARN=$(aws iam create-policy --policy-name $LBC_POLICY_NAME --policy-document file://awsconfigs/infra_configs/iam_alb_ingress_policy.json --output text --query 'Policy.Arn')
+        eksctl create iamserviceaccount --name alb-ingress-controller --namespace kubeflow --cluster ${CLUSTER_NAME} --region ${CLUSTER_REGION} --attach-policy-arn ${LBC_POLICY_ARN} --override-existing-serviceaccounts --approve
+        ```
+1. Configure the parameters for [load balancer controller](../../../../awsconfigs/common/aws-alb-ingress-controller/base/params.env) with the cluster name
+    1. ```
+        printf 'clusterName='$CLUSTER_NAME'' > awsconfigs/common/aws-alb-ingress-controller/base/params.env
+        ```
+
+### Build Manifests and deploy components
+Run the following command to build and install the components specified in this [kustomize](./kustomization.yaml) file.
+```
+kustomize build docs/deployment/add-ons/load-balancer | kubectl apply -f -
+```
+
+### Update the domain with ALB address
+
+1. Check if ALB is provisioned. It takes around 3-5 minutes
+    1. ```
+        kubectl get ingress -n istio-system istio-ingress
+        NAME            CLASS    HOSTS   ADDRESS                                                                  PORTS   AGE
+        istio-ingress   <none>   *       ebde55ee-istiosystem-istio-2af2-1100502020.us-west-2.elb.amazonaws.com   80      15d
```

Comment:
would suggest put dummy address.

Reply:
this is dummy address, replace few chars with xxx to make it more explicit